File: recursion/fibonacci/fibonacci.cpp

#include <iostream>

using namespace std ;

int fib(int v)
{
	if(v == 0 || v == 1)
		return v;
	else
		return fib(v - 1) + fib(v - 2) ;
}

int main(void)
{
	cout << "Fibnacci : " << fib(3) << endl ;
	return 0 ;
}

File: recursion/fibonacci/fibonacci.php


<?php
    function fib($value)
    {
        if($value === 0 || $value === 1)
            return $value ;
        else
            return fib($value - 1) + fib($value - 2) ;
    }

    echo fib(10) . "<br>" ;
?>
